src/main.ts

```diff
@@ -0,0 +1,24 @@
+import "reflect-metadata";
+
+import * as Sentry from "@sentry/node";
+import DIContainer from "./injection/container.injection";
```

Comment:
Why are there no curly brackets around this import?

Reply:
@saltz I am also curious

Reply:
If you refer to the source file "container.injection.ts" you'll see we decided to treat it as the application composition root. Meaning we are exporting an instance of the DIContainer as default (not the module itself) and nothing else.